Modules/IO/MeshGifti/src/itkGiftiMeshIO.cxx

```diff
@@ -140,6 +140,10 @@ GiftiMeshIO::SetLabelNameTable(const LabelNameContainer * labelMap)
   this->Modified();
 }
 
+// TODO:  Need to be able to read/write RGB images into ITK.
```

Comment:
This doesn't look like the right place for this comment.

Reply:
Any objections to just remove this comment?